app/services/ConsignmentService.scala

```diff
@@ -66,6 +68,14 @@ class ConsignmentService @Inject() (val graphqlConfiguration: GraphQLConfigurati
       )
   }
 
+  def areAllFilesClosed(consignment: GetConsignment): Boolean = {
+    consignment.files
+      .filter(file => file.fileMetadata.exists(metadata => metadata.name == "FileType" && metadata.value == "File"))
+      .filter(file => file.fileMetadata.exists(metadata => metadata.name == closureType.name && metadata.value != closureType.value))
+      .map(_.fileId)
```

Comment:
Do you need to `.map` over the files?

Reply:
No I don't good point. I've removed the map and replaced the second filter with exists, thanks.